sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -362,7 +362,10 @@ object DottyPlugin extends AutoPlugin {
 
       // Configuration for the doctool
       resolvers ++= (if(!useScala3doc.value) Nil else Seq(Resolver.jcenterRepo)),
-      useScala3doc := false,
+      useScala3doc := {
+        val v = scalaVersion.value
+        v.startsWith("3.0.0") && !v.startsWith("3.0.0-M1") && !v.startsWith("3.0.0-M2")
```

Comment:
More future-proof:
```suggestion
        v.startsWith("3") && !v.startsWith("3.0.0-M1") && !v.startsWith("3.0.0-M2")
```

Reply:
This setting is going to disappear with 3.0.0, there's no need for this.